### PR TITLE
OO Fixes - Valid PHP Code

### PR DESCRIPTION
**Pull request by nicklog**
> If you have a more concise, less destructive set of specific changes to make with a good argument for them, please submit them as new pull requests.

Here we go again.

Let me know your thougts.

### CODE REVIEW

**Comment by globalcitizen** [comment]
Given the possibility of breaking things for an existing user base, I really don't care about the documentation or stylistic preferences that seem to be the only argument for this complete rewrite of the OO wrapper. Also, did you try using the method parameters? Without testing, I'd wager they probably work.

File: oophp-iban.php [hunk 1]
```diff
@@ -37,15 +37,15 @@ public function HumanFormat() {
   return iban_to_human_format($this->iban);
  }
 
- public function Country($iban='') {
+ public function Country() {
   return iban_get_country_part($this->iban);
```
Comment (acoulton) [inline oophp-iban.php]: @globalcitizen it's pretty clear from the diff of this commit, without any testing required, that those method parameters are not used and are misleading to have in the method signature.

**Comment by acoulton** [comment]
@globalcitizen there's no rewrite here, he's just auto-formatted the code to PSR-2 (which I agree is really not crucial and makes it much harder to visually verify the changes that have actually been made even when it's split to multiple commits).

Per my comment above though, it's fairly obvious the removed method parameters don't in fact work and should be removed, and personally I agree with @nicklog that the added PHPDoc is a good improvement.

**Comment by nicklog** [comment]
Like @acoulton said. There is not rewrite of the code. 
Just fixes for some parts of the code, phpdocs and psr-2 code styling.

> Without testing, I'd wager they probably work.

@globalcitizen the changes passed your test script.

Now I see no argument against merging the source code.
Method names didnt changes and no functions of it changed.

Greetings.

**Pull request closed without merging**